miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppCustomPermissionCache.kt

```diff
@@ -96,44 +96,17 @@ internal class MiniAppCustomPermissionCache(context: Context) {
         applyStoringPermissions(MiniAppCustomPermission(miniAppId, allPermissions))
     }
 
-    @SuppressWarnings("PrintStackTrace")
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     fun applyStoringPermissions(miniAppCustomPermission: MiniAppCustomPermission) {
-        try {
-            val jsonToStore: String = Gson().toJson(orderByDefaultList(miniAppCustomPermission))
-            prefs.edit().putString(miniAppCustomPermission.miniAppId, jsonToStore).apply()
-        } catch (e: Exception) {
-            e.printStackTrace()
-        }
+        val jsonToStore: String = Gson().toJson(sortedByDefault(miniAppCustomPermission))
+        prefs.edit().putString(miniAppCustomPermission.miniAppId, jsonToStore).apply()
     }
 
-    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
-    fun orderByDefaultList(miniAppCustomPermission: MiniAppCustomPermission): MiniAppCustomPermission {
-        val miniAppId = miniAppCustomPermission.miniAppId
-        val defaultTypesOrder = mutableListOf<MiniAppCustomPermissionType>()
-        defaultDeniedList(miniAppId).pairValues.forEach {
-            defaultTypesOrder.add(it.first)
-        }
-
-        val currentTypesOrder = mutableListOf<MiniAppCustomPermissionType>()
-        miniAppCustomPermission.pairValues.forEach {
-            currentTypesOrder.add(it.first)
-        }
-
-        val expectedTypesOrder = currentTypesOrder.map {
-            defaultTypesOrder.indexOf(it)
-        }.sorted().map { value -> defaultTypesOrder[value] }
-
-        val orderedPair =
-            mutableListOf<Pair<MiniAppCustomPermissionType, MiniAppCustomPermissionResult>>()
-
-        expectedTypesOrder.forEachIndexed { index, type ->
-            miniAppCustomPermission.pairValues.find {
-                it.first == type
-            }?.let { orderedPair.add(index, it) }
-        }
-
-        return MiniAppCustomPermission(miniAppId, orderedPair)
+    // Sort the `pairValues` by ordinal of [MiniAppCustomPermissionType].
+    @VisibleForTesting
```

Comment:
`sortedByDefault` function (formerly, `orderByDefaultList`) is using only in `MiniAppCustomPermissionCache`. So, we can make just private. But, to write unit test, we need to use `@VisibleForTesting`.  And, I have used `@VisibleForTesting(otherwise = VisibleForTesting.NONE)` previously to restrict it only for unit test. 

<img width="665" alt="Screen Shot 2020-12-03 at 18 45 43" src="https://user-images.githubusercontent.com/66930373/100992680-d1bb9f00-3597-11eb-8abf-4777608ebc72.png">